### ISSUE TITLE
aws_autoscaling_notification.asd_launch_notify: diffs didn't match during apply.

### ISSUE THREAD

**Issue by Kurry** [hashicorp/terraform#8109]
### Terraform Version

Terraform v0.7.0
### Affected Resource(s)

Please list the resources as a list, for example:
- aws_autoscaling_notification
### Terraform Configuration Files

``` hcl
resource "aws_autoscaling_group" "mqconsumer_asg" {
  name = "mqconsumer_asg-${var.mqconsumer_lc_name}"
  launch_configuration = "${var.mqconsumer_lc_id}"
  load_balancers = ["${var.mqconsumer_elb_name}"]
  vpc_zone_identifier = ["${var.mqconsumer_vpc_zone_identifier}"]
  max_size = "${var.asg_max}"
  min_size = "${var.asg_min}"
  health_check_grace_period = 600
  health_check_type = "ELB"
  force_delete = true
  lifecycle {
    create_before_destroy = true
  }
}
resource "aws_autoscaling_notification" "asd_launch_notify" {
  group_names = [
    "${aws_autoscaling_group.mqconsumer_asg.name}"
  ]
  notifications  = [
    "autoscaling:EC2_INSTANCE_LAUNCH",
    "autoscaling:EC2_INSTANCE_TERMINATE",
    "autoscaling:EC2_INSTANCE_LAUNCH_ERROR"
  ]
  topic_arn = "REMOVED"
}
```
### Debug Output

https://gist.github.com/Kurry/38eed71f3b33388e2793dbc1dfc2e76e
### Expected Behavior

What should have happened?
The program should have created an autoscaling notification.
### Actual Behavior

What actually happened?
The program terminated.
### Steps to Reproduce

Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by radeksimko** [comment]
Hi @Kurry 
thank you for attached debug output, that's helpful.

This is probably the same bug as https://github.com/hashicorp/terraform/issues/4708

**Comment by mitchellh** [comment]
Hello! This should be fixed in later versions of Terraform (0.7.11 at time of writing). 

Its hard to know 100% since there isn't a clear reproduction case here, but we've fixed issues with this exact error message (and type) in recent versions. Please try again. Thanks!
